README.md

```diff
@@ -133,6 +141,30 @@ For example, launching the overlay when a button is clicked
 <button onclick="myUI('pub', 'overlay-show');">Search</button>
 ```
 
+### Input box
+
+Search input config allows you to modify the behaviour of the input component.
+
+| Option | Data | Description |
+| :-- | :-: | :-- |
+| `"mode"` | string | Mode of operation for the input element, can be `standard` or `instant` |
+| `"numSuggestions"` | number | Maximum number of autocomplete suggestions to show |
```

Comment:
Why not `maxSuggestions` (when the documentation begins maximum number as opposed to number, it seems that maximum is somehow significant enough to be included in the actual name?)